dom0/securedrop-check-migration

```diff
@@ -19,11 +19,3 @@ function request_migration() {
     mkdir -p "$flag_dir"
```

Comment:
The `request_migration` function is no longer used; I think this entire file can be deleted?

Reply:
Probably more than that - securedrop-check-migration is used by sd-upgrade-templates, which in turn is running the securedrop-handle-upgrade script, which is also run by sd-remove-unused-templates. There's a whole chunk that can be dropped

Reply:
LGTM but the file also needs to be removed from dom0 spec: https://github.com/freedomofpress/securedrop-workstation/blob/759cbc5686fce192421a00408d00d835439bad65/rpm-build/SPECS/securedrop-workstation-dom0-config.spec#L77